### PR TITLE
To add software beaver habtit tracker

### PR DESCRIPTION
**Pull request by daya0576**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/beaverhabits.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: "Beaver Habit Tracker"
+website_url: "https://github.com/daya0576/beaverhabits"
+source_code_url: "https://github.com/daya0576/beaverhabits"
+description: "A self-hosted habit tracking app to save your precious moments in your fleeting life."
+licenses:
+  - BSD-3-Clause
+platforms:
+  - Python
+  - Docker
```
Comment (Ki-er) [inline software/beaverhabits.yml:9]: > https://github.com/daya0576/beaverhabits?tab=readme-ov-file#self-hosting

```suggestion
platforms:
  - Docker
```

File: software/beaverhabits.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - Time Tracking
+  - Miscellaneous
```
Comment (Ki-er) [inline software/beaverhabits.yml:12]: By having a category, it is inherently not misc so this can be removed. 

```suggestion
tags:
  - Time Tracking
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes to be made.

**Comment by daya0576** [comment]
@Ki-er Thanks a lot for your comments, I have updated my code and am now waiting for the next workflow to verify.

File: software/beaverhabits.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: "Beaver Habit Tracker"
+website_url: "https://github.com/daya0576/beaverhabits"
+source_code_url: "https://github.com/daya0576/beaverhabits"
+description: "A self-hosted habit tracking app to save your precious moments in your fleeting life."
```
Comment (Ki-er) [inline software/beaverhabits.yml:4]:
```suggestion
description: "Habit tracking app to save your precious moments in your fleeting life."
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Another quick change

**Comment by daya0576** [comment]
Thank you, that makes sense as all the software is self-hosted.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**